server/bin/fxa-content-server.js

```diff
@@ -111,7 +111,7 @@ function makeApp() {
 }
 
 var app,
-    port;
+    port; // eslint-disable-line indent
```

Comment:
Let's turn this in to two separate variable declarations. Ideally we'd have a linter that detected this style of variable declarations and warned about it.